pyproject.toml

```diff
@@ -307,6 +310,13 @@ lines-between-types = 1
 order-by-type = false
 
 [tool.ruff.lint.per-file-ignores]
+"docs/additional_examples_and_pseudo_code/**" = [
+  "E501",  # Line too long
+  "E711",  # Comparison to None
```

Comment:
This is an actual code quality issue, we shouldn't be shipping with examples that show bad python coding practices.

Reply:
But it do involves a change in src file.

Reply:
Yep, those files should be changed to show good Python coding standards. This is why our examples should be in python code.

Reply:
This needs to not be ignored, fixing this should not take much time.

Reply:
Since, it includes changes in src file, it needs testing to be done on scopes.
Also, we can't rely on unit -test only.
I understand that the chnage is very minimal, but testing is required.

Reply:
Removing this line should not require changes in src files. This line is only ignoring things in this specific folder. Everything in the src folder should already be checking this and not ignoring it. What src files would need to be changed?